### PR TITLE
Add solana-nodes extension

### PR DESCRIPTION
**Pull request by jstnw10**
## Description

Nodes is a new extension to monitor Solana validators. It displays data in a menu item and within a detailed list view. 

## Screencast

https://github.com/raycast/extensions/assets/40317787/0bad25f7-c742-4659-a546-1c43582a2d7d

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by jstnw10** [comment]
I found some stuff that you maybe would have wanted to be changed, now the extension is publishable as far as I can see.

Happy reviewing!

File: extensions/nodes/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nodes",
+  "title": "Nodes",
```
Comment (pernielsentikaer) [inline extensions/nodes/package.json:4]: Is this the best name for the extension?

File: extensions/nodes/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "menuBarStats",
+      "title": "Solana Validator",
+      "description": "Displays Validator Stats",
+      "mode": "menu-bar",
+      "interval": "1m"
```
Comment (pernielsentikaer) [inline extensions/nodes/package.json:19]: Since this has a static title we should maybe remove the internal since it updates when the user clicks

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

Sorry for the long response time. Due to the Easter vacation, we have a slightly increased response time, but it should soon return to normal.

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/nodes/package.json (hunk 1 above)
Comment (jstnw10) [reply extensions/nodes/package.json:4]: A node is either a RPC or a Validator on Solana, so the name is works fine imo. If you have strong feelings against it, you feel it is misleading or you feel like it should be more descriptive (Solana Nodes, Solana Validator Nodes, ...), pls let me know. 🫡

File: extensions/nodes/package.json (hunk 2 above)
Comment (jstnw10) [reply extensions/nodes/package.json:19]: Gotcha, right. Was looking to show some toast if a certain value changes in the future, didn't implement it yet (and don't know if that is entirely possible). Will remove for now.

**Comment by jstnw10** [comment]
Appreciate the feedback, @pernielsentikaer. I removed the interval and chose to rename the extension to Solana Nodes.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**